.github/workflows/publishDocs.yml

```diff
@@ -0,0 +1,39 @@
+name: publish-docs
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.SVC_CLI_BOT_JSFORCE_GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.SVC_CLI_BOT_JSFORCE_GITHUB_TOKEN }}
+          ref: ${{ github.ref }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+          cache: npm
+      - name: install deps
+        run: npm i
+      - name: build
+        run: npm run clean && npm run build
+      - uses: salesforcecli/github-workflows/.github/actions/gitConfig@main
+      - name: build docs
+        run: |
+          rm -rf docs
+          git worktree prune
+          git fetch origin v2-docs:v2-docs
+          git worktree add docs v2-docs
+          npx typedoc --out docs/tmp
+          cp -r docs/tmp/* docs/tmp/..
```

Comment:
dev-scripts uses `shelljs.mv`:
https://github.com/forcedotcom/dev-scripts/blob/1784d122fce7fd293745a618eee33a98be3476a0/bin/sf-docs.js#L49

shelljs: https://github.com/shelljs/shelljs#mvoptions--source_array-dest

but `mv -f` will not overwrite files so we just copy and then delete them.
https://askubuntu.com/a/269818

Reply:
also, generating docs in `docs/tmp` and then moving them back to `docs` is just to not mess the working tree in `docs`